### DISCUSSION TITLE
Adding italics

### DISCUSSION THREAD

**Discussion by carloterwoord** [ivodolenc/aspekta#19] [Ideas]
### Request Description

Typeface is great! Flexible, distinct, but still simple. Great set of weights, the only thing it misses is are _a great set of italics_ 🤝🏻

### Alternative Solutions

–

**Comment by ivodolenc** [comment]
Hi, thanks for the suggestion, this is something that would definitely be a good addition, but in future releases as this request takes a lot of time, so it's not in focus right now.

I'll mark it as `planned` for future reminders.
